### PR TITLE
[mesheryctl] cleanup golden response files

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Aisuko** [comment]
I really suggest remove these test cases. There are missing understanding for me too. However, I have not got a better solution to test the command line project which was written by using `github.com/spf13/cobra`. Less document in the project.

**Comment by leecalcote** [comment]
Thank you for this, @Aisuko. @Philip-21 @alphaX86 any suggestions on how alternate approaches to testing? 

Wouldn't a golden response file be a copy of the output from the command itself? Or just a couple of words to attest?

**Review by Aisuko — APPROVED** [review]
approve

**Comment by Aisuko** [comment]
I think here is that maybe we can have a xmind to mentions the main features of `mesheryctl`. And trying to re-write it by using more simpler and testable framework.

**Review by alphaX86 — APPROVED** [review]
LGTM

**Comment by Philip-21** [comment]
> Thank you for this, @Aisuko. @Philip-21 @alphaX86 any suggestions on how alternate approaches to testing?
> 
> Wouldn't a golden response file be a copy of the output from the command itself? Or just a couple of words to attest?

Golden files are required for both input and output  such that the `api response.golden`  files are being used as  input when different Mesheryctl commands are tested and the outputs that are generated  in the `output.golden`  files after the api response files have been utilized.

**Review by Philip-21 — APPROVED** [review]

File: mesheryctl/pkg/utils/fixtures/platform/manifesturl.api.response.golden [hunk 1]
```diff
@@ -5010,170 +5010,6 @@
...
-"path": "docs/lib/tasks/templates",
-"mode": "040000",
-"type": "tree",
-"sha": "cfeec97ced6294242a1344b90d1cd5d7c500edf8",
-"url": "https://api.github.com/repos/meshery/meshery/git/trees/cfeec97ced6294242a1344b90d1cd5d7c500edf8"
-},
-{
 "path": "docs/lib/tasks/templates/default.md.haml",
 "mode": "100644",
 "type": "blob",
```
Comment (alphaX86) [inline mesheryctl/pkg/utils/fixtures/platform/manifesturl.api.response.golden:5015]: But still I'm totally surprised that these golden files have entire mock of GitHub API which includes the whole file list of meshery repo. This one must be looked on ＞︿＜

**Pull request merged**